src/components/Footer/Footer.jsx

```diff
@@ -55,4 +64,4 @@ const Footer = () => {
   );
 };
 
-export default Footer;
+export { Footer };
```

Comment:
Footer component is too large on product listing page. Reduce its width.

Reply:
Yeah, will fix that at the end